gns3/main.py

```diff
@@ -184,8 +184,8 @@ def exceptionHook(exception, value, tb):
     if sys.version_info < (3, 4):
         raise SystemExit("Python 3.4 or higher is required")
 
-    if parse_version(QtCore.QT_VERSION_STR) < parse_version("5.0.0"):
-        raise SystemExit("Requirement is PyQt5 version 5.0.0 or higher, got version {}".format(QtCore.QT_VERSION_STR))
+    if parse_version(QtCore.QT_VERSION_STR) < parse_version("5.5.0"):
```

Comment:
I think we will no longer test with version before 5.5 and we suspect they are a source of issues.  All current LTS distribution provide Qt 5.5 or later.